### ISSUE TITLE
Request Log doesn't load in FF4b12pre

### ISSUE THREAD

**Issue by jsamuel** [RequestPolicy/requestpolicy#197]
```
imported trac ticket
created: 2011-02-20 14:31:26
reporter: eibwen
```

The request log in request policy 0.5.18 does not "load" in FF4b12pre Nightly 2011-02-19:

  '''strBundle.getString is not a function'''
  chrome://requestpolicy/content/requestLogTreeView.js

  '''./chrome/content/requestLogTreeView.js'''
  var message = strBundle.getString("requestLogIsEmpty");

The "panel" appears and is close-able; however, there is no treeview.

**Comment by jsamuel** [comment]
```
imported trac comment
created: 2011-02-20 15:43:09
author: eibwen
```

Downgrading priority from the default (highest) to normal.

**Comment by jsamuel** [comment]
```
imported trac comment
created: 2011-02-20 16:04:49
author: justin
```

I can't repeat this with the 4.0b12pre nightly from 2011-02-20. It sounds like #118, which was fixed in RP 0.5.17. I wonder if it's just very infrequent and so I thought it was fixed but it's still possible, just rare?

If you're able to narrow down when it happens and when it doesn't, that would help make it repeatable for me. Otherwise, I'm going to cross my fingers and hope that either it's a temporary Fx beta regression or someone else finds the solution that I can implement and you can verify.

**Comment by jsamuel** [comment]
```
imported trac comment
created: 2011-02-21 23:24:25
author: eibwen
```

Extension conflict:
{{{
RequestPolicy 0.18     and Stylish 1.1
RequestPolicy 0.18     and Stylish 1.1b3
RequestPolicy 0.5.19b2 and Stylish 1.1b3
}}}

RequestPolicy 0.5.19b2 and Stylish 1.01 are compatible; however, Stylish 1.01 is not usable in FF4 (cannot access list of styles to add/edit/change -- integrated into about:addons, which is new in FF4).

RequestPolicy 0.5.19b2 and Stylish 1.0 are compatible; however, Stylish 1.0 is not usable in FF4 due to about:addons...

RequestPolicy 0.5.19b2 and Stylish 0.5.9 are compatible; however, Stylish 0.5.9 does not recognize styles created with later versions...

RequestPolicy 0.5.19b2 and Stylish 0.5 are compatible; however, Stylish 0.5 does not recognize styles created with later versions...

Also affects Adblock+ blockable items...

Also reported at Stylish forum:
http://forum.userstyles.org/discussion/25163/extension-conflict-requestpolicy-adblock
